### ISSUE TITLE
can't enable and disable without reloading window

### ISSUE THREAD

**Issue by smoofra** [streetsidesoftware/vscode-spell-checker#384]
I find that the command "Enable Spell Checking For Workspace" does not work. I run it and nothing happens.   The only way to re-enable spell checking is to edit `.vscode/settings.json` and then reload the window.

**Comment by chronoviking** [comment]
I'm having a similar problem with enable/disable on a file by file basis.

There is an obvious checkmark on the "FILE" tab when looking at settings. But nothing lets me uncheck the box to turn off spell checking. This is frustrating as I'm reading through some debug logs with uses tons of abbreviations, keywords, and slang so the whole right side is covered, hiding any other markers I'm trying to look for (such as found phrase instances).

There should be a way to quickly/easily turn spell checking off and on without having to disable it for a whole filetype or even the whole extension.

**Comment by Jason3S** [comment]
This should be fixed with v1.8.0

**Comment by Jason3S** [comment]
tldr; VS Code had change its API with respect to how updating settings worked. This caused the making changes to the Spell Checker settings to silently fail. This should be fixed now.
